packages/react/src/experimental/Forms/Fields/Select/index.tsx

```diff
@@ -226,9 +238,27 @@ export const Select = forwardRef<HTMLButtonElement, SelectProps<string>>(
             items={items}
             className={selectContentClassName}
             emptyMessage={searchEmptyMessage}
+            bottom={
+              actions && (
+                <>
+                  <div className="flex w-full flex-row gap-2 border-0 border-t border-solid border-f1-border-secondary p-2">
+                    {actions.map((action) => (
+                      <Button
+                        key={action.label}
+                        variant={action.variant}
+                        onClick={action.onClick}
+                        icon={action.icon}
+                        label={action.label}
+                        disabled={action.disabled}
+                      />
+                    ))}
+                  </div>
+                </>
+              )
+            }
```

Comment:
Would you consider to move this to a component with the action logic? or at least in a function?